butler-bot/bot.js

```diff
@@ -26,41 +18,28 @@ client.on('ready', () => {
   logger.info(client.username + ' - (' + client.id + ')')
 })
 
-client.on('guildMemberAdd', (member) => {
-  member.addRole(DESCRIBE_YOURSELF_ROLE_ID)
-    .then(() => { logger.info('Successfully added the role') })
-
-  let greetingsChannel = client.channels.get(GREETINGS_CHANNEL_ID)
-  if (typeof greetingsChannel === 'undefined') {
-    logger.error('Greetings channel is not defined')
-    return
-  }
+const GREETINGS_CHANNEL_ID = '525249065039036426'
 
-  let infoChannel = client.channels.get(INFO_CHANNEL_ID)
-  if (typeof infoChannel === 'undefined') {
-    logger.error('Info channel is not defined')
+client.on('message', msg => {
+  if (msg.channel.id !== GREETINGS_CHANNEL_ID || msg.author.bot) {
     return
   }
 
-  greetingsChannel.send(
-    `Ассаляму алейкум ва рахматуллахи ва баракатуху, ${member.toString()}!\n\n` +
-    `Добро пожаловать на IT-Muslim Дискорд сервер!\n` +
-    `Чтобы узнать побольше о сообществе, посетите канал ${infoChannel.toString()}. ` +
-    `Чтобы получить разрешение писать во всех каналах и активно yчаствовать ` +
-    `в жизни сообщества, напишите прямо сюда (${greetingsChannel.toString()}) ` +
-    `немного о себе.`
-  )
-})
+  let greetingsChannel = client.channels.get(GREETINGS_CHANNEL_ID)
 
-client.on('message', message => {
-  if (message.channel.id !== GREETINGS_CHANNEL_ID || message.author.bot) {
-    return
+  let timeNow = new Date()
+  console.log(timeNow)
+  let timeMemberJoin = msg.member.joinedAt
+  console.log(timeMemberJoin)
+  let timeDifference = (new Date(timeNow).getTime() - new Date(timeMemberJoin).getTime()) / 1000 / 60 / 60 / 24
```

Comment:
It's better to make a separate variable `MILLISECONDS_IN_DAY` and use it.
I also think you could simplify it a bit:
```suggestion
  let timeDifference = (timeNow.getTime() - timeMemberJoin.getTime()) / 1000 / 60 / 60 / 24
```